content/departments/engineering/teams/code-intelligence/index.md

```diff
@@ -10,12 +10,12 @@ The Code Intelligence team builds tools and services that provide contextual inf
 
 - A powerful and flexible language-agnostic model of dependency relationships across projects, repositories, and languages
 
-- Robust, extensible, and scalable infrastructure to index code across all languages, keep those indexes up-to-date, and efficiently resolve code intelligence queries against all indexed code.
+- Robust, extensible, and scalable infrastructure to index code across all languages, keep those indexes up-to-date, and efficiently resolve code navigation queries against all indexed code.
 
 ## Strategy
 
 - We see Code intelligence as the glue that sticks the product together, providing a platform for features from navigation to precise powered searches, compiler-accurate batch changes and insights. We aim to provide support for all widely used languages and for the ones we don’t, provide a platform for any developer to add and test their own indexers.
```

Comment:
@macraig Is this referring to "Code Intelligence" as in the team or the product? Some of these are tricky to change on this page.